tensorflow/tools/compatibility/renames_v2.py

```diff
@@ -1515,6 +1515,10 @@
         'tf.compat.v1.variance_scaling_initializer',
     'tf.verify_tensor_all_finite':
         'tf.compat.v1.verify_tensor_all_finite',
+    'tf.where':
+        'tf.compat.v1.where',
+    'tf.where_v2':
+        'tf.where',
```

Comment:
It cannot be tf.where here because the output of the conversion script has to work in both v1 and v2. Instead this has to say tf.compat.v2.where